### PR TITLE
Planning and Relativemap: Updated with followings in order to use nav…

### PR DESCRIPTION
**Pull request by like6060928**
…igation with utm coor in std model

1. Configs: Added use_relative_map_with_utm to choose whether use navigation with utm coor or hdmap;
2. Hdmap_util: Modified the condition to get relativemap;
3. Navigation_lane: Modified kWeight;
4. Navigation_lane: Transformated utm to flu only in navigaion model with relativemap;
5. Relativemap: Enabled to use relativemap in std model;
6. Std_planning: Modified the condition to use routing and relativemap;
7. Frame: Modified the condition to use routing.
8. Reference_line_provider: Modified the condition to use routing and relativemap;

### CODE REVIEW

File: modules/map/relative_map/BUILD [hunk 1]
```diff
@@ -54,6 +54,7 @@ cc_library(
         "//modules/map/relative_map/proto:navigation_proto",
         "//modules/map/relative_map/proto:relative_map_config_proto",
         "//modules/perception/proto:perception_proto",
+        "//modules/common/configs:config_gflags",
```
Comment (xiaoxq) [inline modules/map/relative_map/BUILD]: Make items in order.

File: modules/map/relative_map/navigation_lane.cc [hunk 2]
```diff
@@ -266,7 +266,12 @@ void NavigationLane::MergeNavigationLineAndLaneMarker(const int line_index,
   }
 
   int lane_marker_index = 0;
-  const double kWeight = 0.9;
+  double weight = 0.0;
+  if (FLAGS_use_navigation_mode) {
+    weight = 0.9;
+  } else {
+    weight = 1.0;
```
Comment (xiaoxq) [inline modules/map/relative_map/navigation_lane.cc]: double weight = FLAGS_use_navigation_mode ? 0.9 : 1.0;

File: modules/planning/reference_line/reference_line_provider.cc [hunk 3]
```diff
@@ -549,19 +549,21 @@ bool ReferenceLineProvider::CreateReferenceLine(
...
-      if (!pnc_map_->UpdateRoutingResponse(routing)) {
-        AERROR << "Failed to update routing in pnc map";
-        return false;
+  if (!FLAGS_use_navigation_with_utm) {
+    routing::RoutingResponse routing;
+    {
+      std::lock_guard<std::mutex> lock(routing_mutex_);
+      routing = routing_;
+    }
+    {
```
Comment (xiaoxq) [inline modules/planning/reference_line/reference_line_provider.cc]: Unnecessary scope.

**Comment by like6060928** [comment]
Thank you for ur advices, I have modified code as required.

**Pull request merged**